### PR TITLE
PXB-2984 8.0.32-25 Release notes and release note tasks (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   docs/release-notes.md
	new file:   docs/release-notes/8.0/8.0.32-25.0.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

**Review by altmannmarcelo — APPROVED** [review]
LGTM

File: docs/release-notes/8.0/8.0.32-25.0.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+c# Percona XtraBackup 8.0.32-25 (2023-02-)
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.0/8.0.32-25.0.md]: Remove c before #

Comment (patrickbirch) [reply docs/release-notes/8.0/8.0.32-25.0.md]: Done.

**Review by satya-bodapati — CHANGES_REQUESTED** [review]
https://github.com/percona/percona-xtrabackup/pull/1411/commits

[PXB-2966 - Merge 8.0.32 (Make Instant stable)](https://github.com/percona/percona-xtrabackup/pull/1411/commits/6e2e54d0a6f8b74a0e88af84ff2b9a19bbad1de6)

We have made a decision to consider INSTANT algorithm stable as of
upstream 8.0.32.

PXB now allows instant columns to be present in the backup.

I think we have to document this. Until now, again Oracle MySQL (not PS), PXB refused to take backups if it finds table with instant columns.

From 8.0.32, we remove this limitation and PXB will take backups of MySQL (8.0.32).
We continue to refuse backups until MySQL 8.0.31 with instant columns.

**Review by alina-derkach-oaza — APPROVED** [review]

**Review by satya-bodapati — APPROVED** [review]

File: docs/release-notes/8.0/8.0.32-25.0.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+[Percona XtraBackup for MySQL Databases](https://www.percona.com/software/mysql-database/percona-xtrabackup) enables MySQL backups without blocking user queries. Percona XtraBackup is ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, Percona XtraBackup drives down backup costs while providing unique features for MySQL backups.
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me).
+
+## Release highlights
+
+This release merges the MySQL 8.0.32 code base.
+
+Percona XtraBackup 8.0.32-25 allows instant columns to be present in the backup for MySQL 8.0.32. The limitation still exists for MySQL 8.0.31 and lower versions.
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.32-25.0.md]:
```suggestion
Starting at Percona XtraBackup 8.0.32-25 allows instant columns to be present in the backup for MySQL 8.0.32. The limitation still exists for backing up MySQL 8.0.31 and lower versions.
The limitation does not exists for backing up Percona Server of any version.
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.0/8.0.32-25.0.md (hunk 2 above)
Comment (patrickbirch) [reply docs/release-notes/8.0/8.0.32-25.0.md]: Rewritten

**Pull request merged**